src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php

```diff
@@ -165,10 +213,35 @@ protected static function createKernel(array $options = array())
             static::$class = static::getKernelClass();
         }
 
-        return new static::$class(
+        if (isset($_SERVER['KERNEL_ENV'])) {
+            $options['environment'] = $_SERVER['KERNEL_ENV'];
+        }
+
+        if (isset($_SERVER['KERNEL_DEBUG'])) {
+            $options['debug'] = $_SERVER['KERNEL_DEBUG'];
```

Comment:
this is broken, because env variables are always strings, while ``debug`` should be a boolean